### ISSUE TITLE
Outdated files in the dev-1.21-ko.5 branch

### ISSUE THREAD

**Issue by seokho-son** [kubernetes/website#28473]
# This is a Bug Report
## Problem
Outdated files in the upstream/dev-1.21-ko.5 branch.
This report is output of `scripts/diff_l10n_branches.py ko upstream/dev-1.21-ko.4 upstream/dev-1.21-ko.5`

### 133 files to be modified 
1. [x] M1.  `content/en/_index.html`  | 3(+XS) 3(-)
1. [X] M2.  `content/en/docs/concepts/architecture/_index.md`  | 0(+XS) 0(-)
1. [x] M3.  `content/en/docs/concepts/architecture/controller.md`  | 4(+XS) 3(-)
1. [x] M4.  `content/en/docs/concepts/cluster-administration/kubelet-garbage-collection.md`  | 31(+M) 19(-)
1. [X] M5.  `content/en/docs/concepts/configuration/_index.md`  | 0(+XS) 0(-)
1. [x] M6.  `content/en/docs/concepts/containers/runtime-class.md`  | 3(+XS) 3(-)
1. [X] M7.  `content/en/docs/concepts/overview/_index.md`  | 0(+XS) 0(-)
1. [X] M8.  `content/en/docs/concepts/overview/working-with-objects/_index.md`  | 0(+XS) 0(-)
1. [X] M9.  `content/en/docs/concepts/overview/working-with-objects/field-selectors.md`  | 1(+XS) 1(-)
1. [x] M10.  `content/en/docs/concepts/policy/_index.md`  | 0(+XS) 0(-)
1. [x] M11.  `content/en/docs/concepts/policy/pod-security-policy.md`  | 6(+XS) 6(-)
1. [x] M12.  `content/en/docs/concepts/policy/resource-quotas.md`  | 18(+S) 12(-)
1. [x] M13.  `content/en/docs/concepts/scheduling-eviction/resource-bin-packing.md`  | 27(+S) 26(-)
1. [x] M14.  `content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md`  | 3(+XS) 3(-)
1. [X] M15.  `content/en/docs/concepts/services-networking/_index.md`  | 0(+XS) 0(-)
1. [x] M16.  `content/en/docs/concepts/services-networking/dns-pod-service.md`  | 3(+XS) 12(-)
1. [x] M17.  `content/en/docs/concepts/services-networking/endpoint-slices.md`  | 1(+XS) 0(-)
1. [x] M18.  `content/en/docs/concepts/services-networking/service.md`  | 1(+XS) 1(-)
1. [X] M19.  `content/en/docs/concepts/storage/_index.md`  | 0(+XS) 0(-)
1. [X] M20.  `content/en/docs/concepts/workloads/controllers/_index.md`  | 0(+XS) 0(-)
1. [x] M21.  `content/en/docs/concepts/workloads/controllers/daemonset.md`  | 32(+M) 21(-)
1. [x] M22.  `content/en/docs/concepts/workloads/controllers/garbage-collection.md`  | 1(+XS) 1(-)
1. [x] M23.  `content/en/docs/concepts/workloads/controllers/job.md`  | 1(+XS) 6(-)
1. [x] M24.  `content/en/docs/concepts/workloads/pods/disruptions.md`  | 1(+XS) 1(-)
1. [x] M25.  `content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md`  | 1(+XS) 1(-)
1. [x] M26.  `content/en/docs/contribute/suggesting-improvements.md`  | 2(+XS) 2(-)
1. [x] M27.  `content/en/docs/reference/_index.md`  | 1(+XS) 1(-)
1. [x] M28.  `content/en/docs/reference/access-authn-authz/service-accounts-admin.md`  | 4(+XS) 4(-)
1. [x] M29.  `content/en/docs/reference/command-line-tools-reference/feature-gates.md`  | 1(+XS) 1(-)
1. [x] M30.  `content/en/docs/reference/glossary/annotation.md`  | 0(+XS) 0(-)
1. [X] M31.  `content/en/docs/reference/glossary/certificate.md`  | 0(+XS) 0(-)
1. [X] M32.  `content/en/docs/reference/glossary/cla.md`  | 0(+XS) 0(-)
1. [X] M33.  `content/en/docs/reference/glossary/cloud-controller-manager.md`  | 0(+XS) 0(-)
1. [X] M34.  `content/en/docs/reference/glossary/cloud-provider.md`  | 0(+XS) 0(-)
1. [X] M35.  `content/en/docs/reference/glossary/cluster.md`  | 0(+XS) 0(-)
1. [X] M36.  `content/en/docs/reference/glossary/configmap.md`  | 0(+XS) 0(-)
1. [X] M37.  `content/en/docs/reference/glossary/container-env-variables.md`  | 0(+XS) 0(-)
1. [X] M38.  `content/en/docs/reference/glossary/container.md`  | 0(+XS) 0(-)
1. [X] M39.  `content/en/docs/reference/glossary/contributor.md`  | 0(+XS) 0(-)
1. [X] M40.  `content/en/docs/reference/glossary/controller.md`  | 0(+XS) 0(-)
1. [X] M41.  `content/en/docs/reference/glossary/cronjob.md`  | 0(+XS) 0(-)
1. [X] M42.  `content/en/docs/reference/glossary/customresourcedefinition.md`  | 0(+XS) 0(-)
1. [X] M43.  `content/en/docs/reference/glossary/daemonset.md`  | 0(+XS) 0(-)
1. [X] M44.  `content/en/docs/reference/glossary/deployment.md`  | 0(+XS) 0(-)
1. [X] M45.  `content/en/docs/reference/glossary/docker.md`  | 0(+XS) 0(-)
1. [X] M46.  `content/en/docs/reference/glossary/dynamic-volume-provisioning.md`  | 0(+XS) 0(-)
1. [X] M47.  `content/en/docs/reference/glossary/etcd.md`  | 0(+XS) 0(-)
1. [X] M48.  `content/en/docs/reference/glossary/image.md`  | 0(+XS) 0(-)
1. [X] M49.  `content/en/docs/reference/glossary/index.md`  | 0(+XS) 0(-)
1. [X] M50.  `content/en/docs/reference/glossary/ingress.md`  | 0(+XS) 0(-)
1. [X] M51.  `content/en/docs/reference/glossary/init-container.md`  | 0(+XS) 0(-)
1. [X] M52.  `content/en/docs/reference/glossary/istio.md`  | 0(+XS) 0(-)
1. [X] M53.  `content/en/docs/reference/glossary/job.md`  | 0(+XS) 0(-)
1. [X] M54.  `content/en/docs/reference/glossary/kube-apiserver.md`  | 0(+XS) 0(-)
1. [X] M55.  `content/en/docs/reference/glossary/kube-controller-manager.md`  | 0(+XS) 0(-)
1. [X] M56.  `content/en/docs/reference/glossary/kube-proxy.md`  | 0(+XS) 0(-)
1. [X] M57.  `content/en/docs/reference/glossary/kube-scheduler.md`  | 0(+XS) 0(-)
1. [X] M58.  `content/en/docs/reference/glossary/kubectl.md`  | 0(+XS) 0(-)
1. [X] M59.  `content/en/docs/reference/glossary/kubelet.md`  | 0(+XS) 0(-)
1. [X] M60.  `content/en/docs/reference/glossary/kubernetes-api.md`  | 0(+XS) 0(-)
1. [X] M61.  `content/en/docs/reference/glossary/label.md`  | 0(+XS) 0(-)
1. [X] M62.  `content/en/docs/reference/glossary/limitrange.md`  | 0(+XS) 0(-)
1. [X] M63.  `content/en/docs/reference/glossary/managed-service.md`  | 0(+XS) 0(-)
1. [X] M64.  `content/en/docs/reference/glossary/minikube.md`  | 0(+XS) 0(-)
1. [X] M65.  `content/en/docs/reference/glossary/mirror-pod.md`  | 0(+XS) 0(-)
1. [X] M66.  `content/en/docs/reference/glossary/name.md`  | 0(+XS) 0(-)
1. [X] M67.  `content/en/docs/reference/glossary/namespace.md`  | 0(+XS) 0(-)
1. [X] M68.  `content/en/docs/reference/glossary/network-policy.md`  | 0(+XS) 0(-)
1. [X] M69.  `content/en/docs/reference/glossary/node.md`  | 0(+XS) 0(-)
1. [X] M70.  `content/en/docs/reference/glossary/object.md`  | 0(+XS) 0(-)
1. [X] M71.  `content/en/docs/reference/glossary/persistent-volume-claim.md`  | 0(+XS) 0(-)
1. [X] M72.  `content/en/docs/reference/glossary/persistent-volume.md`  | 0(+XS) 0(-)
1. [X] M73.  `content/en/docs/reference/glossary/pod-security-policy.md`  | 0(+XS) 0(-)
1. [X] M74.  `content/en/docs/reference/glossary/pod.md`  | 0(+XS) 0(-)
1. [X] M75.  `content/en/docs/reference/glossary/qos-class.md`  | 0(+XS) 0(-)
1. [X] M76.  `content/en/docs/reference/glossary/rbac.md`  | 0(+XS) 0(-)
1. [X] M77.  `content/en/docs/reference/glossary/replica-set.md`  | 0(+XS) 0(-)
1. [X] M78.  `content/en/docs/reference/glossary/replication-controller.md`  | 0(+XS) 0(-)
1. [X] M79.  `content/en/docs/reference/glossary/resource-quota.md`  | 0(+XS) 0(-)
1. [X] M80.  `content/en/docs/reference/glossary/secret.md`  | 0(+XS) 0(-)
1. [X] M81.  `content/en/docs/reference/glossary/selector.md`  | 0(+XS) 0(-)
1. [X] M82.  `content/en/docs/reference/glossary/service-account.md`  | 0(+XS) 0(-)
1. [X] M83.  `content/en/docs/reference/glossary/service-broker.md`  | 0(+XS) 0(-)
1. [X] M84.  `content/en/docs/reference/glossary/service-catalog.md`  | 0(+XS) 0(-)
1. [X] M85.  `content/en/docs/reference/glossary/service.md`  | 0(+XS) 0(-)
1. [X] M86.  `content/en/docs/reference/glossary/statefulset.md`  | 0(+XS) 0(-)
1. [X] M87.  `content/en/docs/reference/glossary/static-pod.md`  | 0(+XS) 0(-)
1. [X] M88.  `content/en/docs/reference/glossary/storage-class.md`  | 0(+XS) 0(-)
1. [X] M89.  `content/en/docs/reference/glossary/uid.md`  | 0(+XS) 0(-)
1. [X] M90.  `content/en/docs/reference/glossary/volume-plugin.md`  | 0(+XS) 0(-)
1. [X] M91.  `content/en/docs/reference/glossary/volume.md`  | 0(+XS) 0(-)
1. [X] M92.  `content/en/docs/reference/kubectl/_index.md`  | 0(+XS) 0(-)
1. [X] M93.  `content/en/docs/reference/setup-tools/kubeadm/_index.md`  | 0(+XS) 0(-)
1. [x] M94.  `content/en/docs/reference/using-api/client-libraries.md`  | 1(+XS) 0(-)
1. [x] M95.  `content/en/docs/setup/_index.md`  | 29(+S) 5(-)
1. [x] M96.  `content/en/docs/setup/best-practices/cluster-large.md`  | 1(+XS) 1(-)
1. [X] M97.  `content/en/docs/tasks/access-application-cluster/_index.md`  | 0(+XS) 0(-)
1. [x] M98.  `content/en/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md`  | 21(+S) 22(-)
1. [x] M99.  `content/en/docs/tasks/access-application-cluster/list-all-running-container-images.md`  | 4(+XS) 4(-)
1. [X] M100.  `content/en/docs/tasks/administer-cluster/_index.md`  | 0(+XS) 0(-)
1. [x] M101.  `content/en/docs/tasks/administer-cluster/change-pv-reclaim-policy.md`  | 1(+XS) 1(-)
1. [X] M102.  `content/en/docs/tasks/administer-cluster/kubeadm/_index.md`  | 0(+XS) 0(-)
1. [x] M103.  `content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md`  | 1(+XS) 1(-)
1. [x] M104.  `content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md`  | 5(+XS) 1(-)
1. [X] M105.  `content/en/docs/tasks/configure-pod-container/_index.md`  | 0(+XS) 0(-)
1. [x] M106.  `content/en/docs/tasks/configure-pod-container/configure-runasusername.md`  | 2(+XS) 2(-)
1. [x] M107.  `content/en/docs/tasks/configure-pod-container/pull-image-private-registry.md`  | 1(+XS) 1(-)
1. [x] M108.  `content/en/docs/tasks/configure-pod-container/quality-service-pod.md`  | 6(+XS) 4(-)
1. [X] M109.  `content/en/docs/tasks/debug-application-cluster/_index.md`  | 0(+XS) 0(-)
1. [x] M110.  `content/en/docs/tasks/debug-application-cluster/determine-reason-pod-failure.md`  | 1(+XS) 1(-)
1. [X] M111.  `content/en/docs/tasks/inject-data-application/_index.md`  | 0(+XS) 0(-)
1. [X] M112.  `content/en/docs/tasks/manage-daemon/_index.md`  | 0(+XS) 0(-)
1. [x] M113.  `content/en/docs/tasks/manage-daemon/update-daemon-set.md`  | 12(+S) 17(-)
1. [x] M114.  `content/en/docs/tasks/manage-kubernetes-objects/kustomization.md`  | 3(+XS) 3(-)
1. [X] M115.  `content/en/docs/tasks/network/_index.md`  | 0(+XS) 0(-)
1. [X] M116.  `content/en/docs/tasks/run-application/_index.md`  | 0(+XS) 0(-)
1. [X] M117.  `content/en/docs/tasks/tls/_index.md`  | 0(+XS) 0(-)
1. [X] M118.  `content/en/docs/tasks/tools/_index.md`  | 0(+XS) 0(-)
1. [x] M119.  `content/en/docs/tasks/tools/install-kubectl-linux.md`  | 5(+XS) 0(-)
1. [x] M120.  `content/en/docs/tasks/tools/install-kubectl-macos.md`  | 4(+XS) 6(-)
1. [x] M121.  `content/en/docs/tasks/tools/install-kubectl-windows.md`  | 1(+XS) 6(-)
1. [x] M122.  `content/en/docs/tutorials/_index.md`  | 1(+XS) 1(-)
1. [X] M123.  `content/en/docs/tutorials/clusters/_index.md`  | 0(+XS) 0(-)
1. [X] M124.  `content/en/docs/tutorials/configuration/_index.md`  | 0(+XS) 0(-)
1. [x] M125.  `content/en/docs/tutorials/configuration/configure-java-microservice/_index.md`  | 0(+XS) 0(-)
1. [x] M126.  `content/en/docs/tutorials/hello-minikube.md`  | 1(+XS) 1(-)
1. [X] M127.  `content/en/docs/tutorials/services/_index.md`  | 0(+XS) 0(-)
1. [X] M128.  `content/en/docs/tutorials/stateful-application/_index.md`  | 0(+XS) 0(-)
1. [x] M129.  `content/en/docs/tutorials/stateful-application/basic-stateful-set.md`  | 3(+XS) 3(-)
1. [X] M130.  `content/en/docs/tutorials/stateless-application/_index.md`  | 0(+XS) 0(-)
1. [x] **M131.**  `content/en/docs/tutorials/stateless-application/guestbook.md`  | **138(+L) 66(-)**
1. [x] M132.  `content/en/examples/application/guestbook/frontend-deployment.yaml`  | 12(+S) 15(-)
1. [x] M133.  `content/en/examples/application/guestbook/frontend-service.yaml`  | 7(+XS) 4(-)

### 1 files to be renamed  
1. [x] R1.  R057 `content/en/docs/tasks/administer-cluster/highly-available-master.md` -> `content/en/docs/tasks/administer-cluster/highly-available-control-plane.md`

### 3 files to be deleted 
1. [x] D1.  `content/en/docs/tasks/tools/included/install-kubectl-gcloud.md`
1. [x] D2.  `content/en/examples/application/guestbook/mongo-deployment.yaml`
1. [x] D3.  `content/en/examples/application/guestbook/mongo-service.yaml`

## Proposed Solution

Use `git diff` to check what is changed in the upstream. And apply the upstream changes manually 
to the `content/ko` of `upstream/dev-1.21-ko.5` branch.

For example:
```
# checkout `upstream/dev-1.21-ko.5`
...
# check what is updated in the upstream 
git diff upstream/dev-1.21-ko.4 upstream/dev-1.21-ko.5 -- content/en/_index.html
# apply changes to content/ko
vi content/ko/_index.html
...
# commit and push
...
# make PR to `upstream/dev-1.21-ko.5`
```

## Pages to Update

Pages in content/ko

**Comment by seokho-son** [comment]
@gochist 
0(+XS) 0(-) 라인 변화가 없는데 포함된 파일들은 아마도 파일 권한 변경이 표시된 것이 아닌가 합니다.

**Comment by jihoon-seo** [comment]
> 0(+XS) 0(-) 라인 변화가 없는데 포함된 파일들은 아마도 파일 권한 변경이 표시된 것이 아닌가 합니다.

네 맞습니다. 😊 (#28447)

**Comment by jihoon-seo** [comment]
퍼미션 관련은 제가 처리하겠습니다. 😊
/assign

**Comment by jihoon-seo** [comment]
> ### 3 files to be modified
> 131. [ ]  **M131.**  `content/en/docs/tutorials/stateless-application/guestbook.md`  | **138(+L) 66(-)**
> 132. [ ]  M132.  `content/en/examples/application/guestbook/frontend-deployment.yaml`  | 12(+S) 15(-)
> 133. [ ]  M133.  `content/en/examples/application/guestbook/frontend-service.yaml`  | 7(+XS) 4(-)
> 
> ### 2 files to be deleted
> 2. [ ]  D2.  `content/en/examples/application/guestbook/mongo-deployment.yaml`
> 3. [ ]  D3.  `content/en/examples/application/guestbook/mongo-service.yaml`

제가 하겠습니다~~

**Comment by ClaudiaJKang** [comment]
M1~M20까지 제가 하겠습니다~

**Comment by jihoon-seo** [comment]
M21~M29 제가 하겠습니다~~

**Comment by jihoon-seo** [comment]
M94 ~ M110, R057 제가 하겠습니다~~

**Comment by jihoon-seo** [comment]
M113 ~ M129 (이 중에서 M122 는 다른 PR에서 다루었음)
제가 하겠습니다~~

(혹시 하시고 싶은 분이 있으시면
저 대신 맡아서 하셔도 좋을 것 같습니다 😊)

**Comment by seokho-son** [comment]
137개 task 처리 모두 확인하였습니다. 감사합니다~~~ @jihoon-seo @ClaudiaJKang

**Comment by jihoon-seo** [comment]
This issue was resolved by the PR #28739 and so many other PRs.
/close
